buildSrc/src/main/kotlin/JsPlugin.kt

```diff
@@ -58,16 +58,16 @@ abstract class JsPlugin : Plugin<Project> {
         target.pluginManager.apply(NodePlugin::class.java)
         target.extensions.configure(NodeExtension::class.java) {
             download = true
-            version = Versions.node
+            version = "12.2.0"
```

Comment:
Shouldn't we use property?

Reply:
I am not sure. Probably we won't check and upgrade this version, and this code will be removed when Kotlin 1.3.40 will be ready.